Documentation/ExtensionArchitecture/Tutorials/Tea/DirectoryStructure.rst

```diff
@@ -0,0 +1,189 @@
+..  include:: /Includes.rst.txt
+
+..  index::
+    Tutorial Tea; Directory structure
+..  _extbase_tutorial_tea_extension_configuration:
+
+============================
+Create a directory structure
+============================
+
+Extbase requires or defaults to a certain directory structure. It is considered
+best practise to always stick to this structure.
+
+On the first level EXT:tea has the following structure:
+
+..  code-block:: none
+    :caption: Directory structure of EXT:tea
+
+    $ tree typo3conf/ext/tea
```

Comment:
We should avoid `typo3conf/ext` as this is gone for Composer-based extensions using typo3/cms-composer-installers v4 (available as RC for TYPO3 v11 and AFAIK v12 relies on it). The extension is then installed under `vendors/ttn/tea` with a symlink or `Resources/Public` to `_assets/<some hash>/`